source/installation/resource-manager/kubernetes.rst

```diff
@@ -265,13 +319,87 @@ for more details.
 Kyverno Policies
 ****************
 
+Once Kubernetes is available to OnDemand, it's possible for users to use ``kubectl`` to submit arbirary pods to Kubernetes. To ensure proper security with Kubernetes a policy engine such as `Kyverno`_ can be used to ensure certain security standards.
+
+For OnDemand maybe of the `Kyverno baseline and restricted sescurity policies`_ will work.  There are also policies that can be deployed to ensure the UID/GID of user pods match that user's UID/GID on the HPC clusters.  Some `example policies`_ do things such as enforce UID/GID and other security standards for OnDemand. These policies rely heavily on the fact that OnDemand usage in Kubernetes using a namespace prefix.
```

Comment:
```suggestion
For OnDemand, maybe some of the `Kyverno baseline and restricted sescurity policies`_ will work.  There are also policies that can be deployed to ensure the UID/GID of user pods match that user's UID/GID on the HPC clusters.  Some `example policies`_ do things such as enforce UID/GID and other security standards for OnDemand. These policies rely heavily on the fact that OnDemand usage in Kubernetes using a namespace prefix.
```

Reply:
I will add the comma, I pushed a fix as `maybe` was a typo, should have been `For OnDemand, many of the`.

Reply:
:+1: